framework/src/base/Moose.C

```diff
@@ -1193,3 +1193,5 @@ bool _deprecated_is_error = false;
 bool _throw_on_error = false;
 
 } // namespace Moose
+
+OStreamProxy & _console = Moose::out;
```

Comment:
This is another good answer saying [it's guaranteed](http://stackoverflow.com/questions/8784892/is-stdcout-guaranteed-to-be-initialized) to work for C++11 and worked in practice for all C++03 compilers.